config/locales/en.yml

```diff
@@ -3,7 +3,10 @@ en:
   activerecord:
     errors:
       models:
-        # employee:
+        employee:
+          attributes:
+              base:
+                department_has_20_employees: "This department already has 20 employees"
```

Comment:
In case you need to change the limit to 30, you should need to change a lot of places. Let's just leave it like: `department_employees_limit`